OpenRA.Platforms.Default/OpenAlSoundEngine.cs

```diff
@@ -286,6 +287,11 @@ public void SetListenerPosition(WPos position)
 			AL.Listener(ALListenerfv.Orientation, ref orientation);
 			AL.Listener(ALListenerf.EfxMetersPerUnit, .01f);
 		}
+
+		public void Dispose()
+		{
+			Alc.CloseDevice(dev);
```

Comment:
Can you check for ```IntPtr.Zero``` here to prevent invalid frees. Also set ```IntPtr.Zero``` once you free it to prevent double frees.

Seal this class, too.